plot_krun_results.py

```diff
@@ -663,6 +706,17 @@ def create_cli_parser():
                         dest='sigma',
                         default=False,
                         help='Draw 5-sigma interval around the rolling mean.')
+    parser.add_argument('--inset-limit',
+                        action='store',
+                        dest='inset_xlimit',
+                        default=None,
+                        type=int,
+                        metavar='LIMIT',
+                        help='Place a small chart plotting the first LIMIT '
```

Comment:
Should `--inset-(x)limit` work like `--xlimit`, i.e. a 2-tuple of (start, end). This would allow us to zoom in on arbitrary interesting events on a graph.